spec/system/candidate_interface/candidate_entering_other_qualification_spec.rb

```diff
@@ -150,4 +161,33 @@ def when_i_change_my_qualification
   def then_i_can_check_my_revised_qualification
     expect(page).to have_content 'A-Level How to Win Against Kaiba'
   end
+
+  def when_i_click_on_continue
+    click_button t('application_form.other_qualification.review.button')
+  end
+
+  def then_i_should_see_the_form_and_the_section_is_not_completed
```

Comment:
not sure, should we split this? or maybe just renaming this method with something shorter and meaningful?
`then_i_should_see_the_page_ ... `

Reply:
Have split this!